_health-care/_js/components/personal-information/VeteranAddressSection.jsx

```diff
@@ -1,8 +1,24 @@
 import React from 'react';
 
 import Phone from './Phone';
+import Email from './Email';
 
 class VeteranAddressSection extends React.Component {
+  constructor() {
+    super();
+    this.confirmEmail = this.confirmEmail.bind(this);
+  }
+
+  confirmEmail() {
+    let errorMessage = undefined;
+
+    if (this.props.data.email !== this.props.data.emailConfirmation) {
+      errorMessage = 'Please ensure your entries match';
```

Comment:
Don't use a variable here.  Just `return 'Please ensure...';` in this branch and `return undefined;` at the bottom. Fewer variables == easier to read.